src/components/Board/Board.actions.js

```diff
@@ -272,8 +272,9 @@ export function previousBoard() {
 }
 
 export function toRootBoard() {
-  return {
-    type: TO_ROOT_BOARD
+  return dispatch => {
+    history.replace('/board/root');
```

Comment:
It should not redirect to `/board/root` , which is the default board. Instead, it should redirect to the user's root board.